README.md

```diff
@@ -144,7 +144,7 @@ The Sending Chain of the Requestor MUST always match the Receiving Chain of the
       │                             │                │
       │                             │                │
       │                             ▼                │
-      │                        ┌─────────┐  512-603  │
+      │                        ┌─────────┐  512-607  │
```

Comment:
"Addition: how does it work?"

Reply:
lol, i notice this but was like @expede would never get this wrong so i must be missing something because *addition* is such a complex operation :) bahh